api/test/backup.test.ts

```diff
@@ -61,16 +65,19 @@ describe('Backup and restore', () => {
       }
       expect(count).to.equal(17);
 
-      // throw in a test of getNotebookRecords while we're here
-      const records = await getNotebookRecords(
+      // throw in a test of getRecordsWithRegex while we're here
+      const token = {
+        roles: user.roles,
+        server: slugify(CONDUCTOR_INSTANCE_NAME),
+        username: user.user_id,
+        // Five minutes from now
+        exp: Date.now() + 1000 * 60 * 5,
+      };
```

Comment:
maybe we could make this a util function for testing/ i.e. buildUserToken or similar